lib/xlint.rb

```diff
@@ -28,6 +29,8 @@ def check_env
 
     def build_draft
       @comments = []
+      # GitDiffParser::Patches.parse(cp932 text) raises ArgumentError: invalid byte sequence in UTF-8
```

Comment:
When does this error happen? Is there another diff parser we could use that's more robust?